scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -165,10 +165,10 @@ function doAddPlugin() {
   then
     if [ "${1}" = "startexplorer" ]
     then
-      doInstallEclipsePlugin "de.bastiankrol.startexplorer.feature.feature.group" "http://basti1302.github.com/startexplorer/update/"
```

Comment:
couldn't we reuse here the properties instead of respecifying the hole stuff?
I am not getting why this code is here again whereas we have the same configuration in the plugin properties

Reply:
You need to understand the history of `devonfw-ide`. We need to care about downward compatibility.
The `plugins` folder with the properties files was later introduced. However, we still have users that never had and still do not have this folder in their settings git.
Removing this code will break their IDE installation.